### ISSUE TITLE
PaginationActions not exported from react-admin

### ISSUE THREAD

**Issue by glauff** [marmelab/react-admin#4926]
**What you were expecting:**
I expected to import PaginationActions from react-admin as advised in react-admin Documentation:
https://marmelab.com/react-admin/List.html

**What happened instead:**
Failed to compile.
Attempted import error: 'PaginationActions' is not exported from 'react-admin' (imported as 'RaPaginationActions').

**Steps to reproduce:**
Follow the instructions in react-admin List Documentation on Pagination

**Related code:**

```
import {
    List,
    Pagination as RaPagination,
    PaginationActions as RaPaginationActions,
} from 'react-admin';

export const PaginationActions = props => <RaPaginationActions {...props} color="secondary" />;

export const Pagination = props => <RaPagination {...props} ActionsComponent={PaginationActions} />;

export const UserList = props => (
    <List {...props} pagination={<Pagination />}>
    </List>
);
```

**Environment**

* React-admin version: 3.6.0

**Comment by fzaninotto** [comment]
You're right, thanks for the report.

**Comment by fzaninotto** [comment]
Sorry, I push the fix on `master` by mistake. The fix will be released in version 3.6.1.
